domain/consensus/utils/txscript/data/script_tests.json

```diff
@@ -4847,12 +4733,6 @@
     "SIG_PUSHONLY",
     "Tests for Script.IsPushOnly()"
   ],
-  [
-    "NOP1 0x01 1",
-    "BLAKE2B 0x20 0xda1745e9b549bd0bfa1a569971c77eba30cd5a4b EQUAL",
```

Comment:
Why did you remove it?

Reply:
Yes, because we have this just before this
```
  [
    "NOP 0x01 1",
    "BLAKE2B 0x20 0xda1745e9b549bd0bfa1a569971c77eba30cd5a4b EQUAL",
    "",
    "SIG_PUSHONLY",
    "Tests for Script.IsPushOnly()"
  ],
```